tests/test_dataclasses.py

```diff
@@ -450,41 +450,41 @@ def test_csv_fields():
                 ("creation_time", "rate [ct./kWh]", "energy [kWh]", "price [ct.]", "buyer"))
 
 
-class TestTradeBidOfferInfo:
+class TestTradeOrdersInfo:
     @staticmethod
     def test_to_json_string():
-        trade_bid_offer_info = TradeBidOfferInfo(
+        trade_orders_info = TradeOrdersInfo(
             1, 1, 1, 1, 1
         )
-        assert (trade_bid_offer_info.to_json_string() ==
-                json.dumps(asdict(trade_bid_offer_info), default=json_datetime_serializer))
+        assert (trade_orders_info.to_json_string() ==
+                json.dumps(asdict(trade_orders_info), default=json_datetime_serializer))
 
     @staticmethod
     def test_from_json():
-        trade_bid_offer_info = TradeBidOfferInfo(
+        trade_orders_info = TradeOrdersInfo(
             1, 1, 1, 1, 1
         )
-        trade_bid_offer_info_json = trade_bid_offer_info.to_json_string()
-        assert (TradeBidOfferInfo.from_json(trade_bid_offer_info_json) ==
-                trade_bid_offer_info)
+        trade_orders_info_json = trade_orders_info.to_json_string()
+        assert (TradeOrdersInfo.from_json(trade_orders_info_json) ==
+                trade_orders_info)
 
 
 class TestTrade:
     def setup_method(self):
         self.initial_data = {
             "id": "my_id",
             "creation_time": DEFAULT_DATETIME,
-            "offer_bid": Offer("id", DEFAULT_DATETIME, 1, 2, "seller"),
+            "order": Offer("id", DEFAULT_DATETIME, 1, 2, "seller"),
             "seller": "seller",
             "buyer": "buyer"}
 
     def test_str(self):
         trade = Trade(**self.initial_data)
         assert (str(trade) ==
                 f"{{{trade.id!s:.6s}}} [origin: {trade.seller_origin} -> {trade.buyer_origin}] "
-                f"[{trade.seller} -> {trade.buyer}] {trade.offer_bid.energy} kWh"
-                f" @ {trade.offer_bid.price} {round(trade.offer_bid.energy_rate, 8)} "
-                f"{trade.offer_bid.id} [fee: {trade.fee_price} cts.]")
+                f"[{trade.seller} -> {trade.buyer}] {trade.order.energy} kWh"
+                f" @ {trade.order.price} {round(trade.order.energy_rate, 8)} "
```

Comment:
Please use `limit_float_precision ` as well as in other occurrences too.

Reply:
I'm afraid that I can't do it in the context of this PR, there are some round calls that round by 8 and others by 4 inside the same module (and I suppose some of them are used in results calculations)
I think this is to be refactored indeed, but I don't believe that this is the right place for it (these PRs were originally created to only deal with one term and they're already carrying lots of changes)